tests.md

```diff
@@ -0,0 +1,238 @@
+# Tests
+
+The WordPress Hosting Team provides tools for hosting companies to run the WordPress automated tests on their infrastructure to improve compatibility with WordPress. These results can be published them on the [Host Test Result information page](https://make.wordpress.org/hosting/test-results/), to help WordPress' compatibility with hosts as well.
+
+It consists of two tools: the Runner, which is part of the core's PHPUnit tests, which work on a hosting, and optionally [sends the information to the results page](https://make.wordpress.org/hosting/test-results/); and the Reporter, which is the plugin that [works on the hosting page](https://make.wordpress.org/hosting/) and shows the results.
```

Comment:
"the Runner, which is part of the core's PHPUnit tests, which work on a hosting" =>
"the runner is the part that runs core's PHPUnit tests on a host" (or "web host")